test/ServiceManagerTest.php

```diff
@@ -475,7 +487,6 @@ public function testConfigureMultipleTimesAvoidsDuplicates(): void
         ];
 
         $serviceManager = new ServiceManager($config);
-        $serviceManager->configure($config);
```

Comment:
This test is explicitly called `testConfigureMultipleTimesAvoidsDuplicates` - it's designed to do the configuration multiple times, so this call should stay